datasync/chaindatafetcher/kas/repository_transactions.go

```diff
@@ -111,9 +111,12 @@ func transformToTxs(event blockchain.ChainEvent) []*Tx {
 }
 
 // InsertTransactions inserts transactions in the given chain event into KAS database.
-// The transactions are divided into chunkUnit because of max number of placeholders.
 func (r *repository) InsertTransactions(event blockchain.ChainEvent) error {
-	txs := transformToTxs(event)
+	return r.insertTransactions(transformToTxs(event))
+}
+
+// insertTransactions inserts the given transactions divided into chunkUnit because of the max number of placeholders.
+func (r *repository) insertTransactions(txs []*Tx) error {
```

Comment:
Isn't this the one I've commented in the previous PR? https://github.com/klaytn/klaytn/pull/583#discussion_r464865820

Reply:
Yes, I divided into this because internal transaction is inserted into `klay_transfers` table as well. So, this is necessary.